packages/eslint-plugin/test/__file_snapshots__/types/no-import-of-dev-dependencies/bad.d.ts.lint

```diff
@@ -1,17 +1,23 @@
 types/no-import-of-dev-dependencies/bad.d.ts
-  1:21  error  .d.ts files may not triple-slash reference packages in devDependencies  @definitelytyped/no-import-of-dev-dependencies
-  2:21  error  .d.ts files may not triple-slash reference packages in devDependencies  @definitelytyped/no-import-of-dev-dependencies
-  4:1   error  .d.ts files may not import packages in devDependencies                  @definitelytyped/no-import-of-dev-dependencies
-  5:1   error  .d.ts files may not import packages in devDependencies                  @definitelytyped/no-import-of-dev-dependencies
+  1:1   error  Do not use a triple slash reference for devdep, use `import` style instead       @typescript-eslint/triple-slash-reference
+  1:21  error  .d.ts files may not triple-slash reference packages in devDependencies           @definitelytyped/no-import-of-dev-dependencies
+  2:1   error  Do not use a triple slash reference for otherdevdep, use `import` style instead  @typescript-eslint/triple-slash-reference
+  2:21  error  .d.ts files may not triple-slash reference packages in devDependencies           @definitelytyped/no-import-of-dev-dependencies
+  4:1   error  .d.ts files may not import packages in devDependencies                           @definitelytyped/no-import-of-dev-dependencies
+  5:1   error  .d.ts files may not import packages in devDependencies                           @definitelytyped/no-import-of-dev-dependencies
 
-✖ 4 problems (4 errors, 0 warnings)
+✖ 6 problems (6 errors, 0 warnings)
 
 ==== types/no-import-of-dev-dependencies/bad.d.ts ====
 
     /// <reference types="devdep"/>
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+!!! @typescript-eslint/triple-slash-reference: Do not use a triple slash reference for devdep, use `import` style instead.
```

Comment:
I assume that this only appears because I also import that same name from this file below, and the rule is doing what we want otherwise?

Reply:
Yeah, the rule is working as expected, and I thought it was better to show the error than to add a suppression comment. Let me know if you want it the other way round, though.

Reply:
Nah, I'd leave it. Nice to have the baseline IMO